src/executor/query/UnwindExecutor.cpp

```diff
@@ -0,0 +1,65 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "executor/query/UnwindExecutor.h"
+#include "context/QueryExpressionContext.h"
+#include "parser/Clauses.h"
+#include "planner/Query.h"
+#include "util/ScopedTimer.h"
+
+namespace nebula {
+namespace graph {
+
+folly::Future<Status> UnwindExecutor::execute() {
+    SCOPED_TIMER(&execTime_);
+
+    auto *unwind = asNode<Unwind>(node());
+    auto *expr = const_cast<Expression *>(unwind->expr());
+
+    auto iter = ectx_->getResult(unwind->inputVar()).iter();
+    DCHECK(!!iter);
+    QueryExpressionContext ctx(ectx_);
+
+    DataSet ds;
+    ds.colNames = unwind->colNames();
+    if (expr->kind() == Expression::Kind::kVarProperty) {   // UNWIND var AS r
```

Comment:
Are you sure the expression kind supported only contains `VarProperty` and `ConstantExpression` ? How do you handle the scenario like `var[0]`.